src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.DataBox</PackageId>
+		<Description>Microsoft Azure Management DataBox library</Description>
+		<AssemblyName>Microsoft.Azure.Management.DataBox</AssemblyName>
+		<Version>1.0.0-preview</Version>
+		<PackageTags>Microsoft Azure DataBox management;DataBox;DataBox management;</PackageTags>
+		<NoWarn>$(NoWarn);CS1591</NoWarn>
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+		<RootNamespace>Microsoft.Azure.Management.DataBox</RootNamespace>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
@kushalvms can you remove this. We do nto want to create pacakges on build. We have separate targets for creating pakcages, this has direct impact on CI perf.
msbuild build.proj /t:CreateNugetPackage /p:Scope=SDKs\DataBox